packages/core/src/corelib.d.ts

```diff
@@ -404,6 +404,18 @@ interface Array<T> {
      * @param separator A string used to separate one element of the array from the next in the resulting string. If omitted, the array elements are separated with a comma.
      */
     join(separator?: string): string
+
+    /**
+     * Sorts an array in place.
+     * This method mutates the array and returns a reference to the same array.
```

Comment:
Maybe add a comment about using insertion sort (n^2 complexity) ?

Reply:
Added a comment that explains the method and complexity